app/javascript/components/DownloadButton.js

```diff
@@ -1,25 +1,21 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDownload } from '@fortawesome/free-solid-svg-icons';
 import Modal from 'react-bootstrap/lib/Modal';
+import Clipboard from 'react-clipboard.js';
 
-// TODO: Remove this once API endpoint is integrated
-import {authCodeResponseMock} from './FacetsMockData';
+import { fetchAuthCode } from './../lib/scp-api';
 
```

Comment:
The name 'fetchDownloadConfig threw me off -- I was expecting it to fetch a download config object from the server.  But it looks like it's fetching an authCode and then assembling a curl command from that?  So maybe call it 'generateDownloadConfigCommand' ?

Reply:
Addressed in 034ff9d.